aprotos/07_CSS2/hw_5flex.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
This task is not finished

Reply:
now finished
used other student's work as a exmaple